pylint/extensions/docparams.py

```diff
@@ -589,7 +589,7 @@ class constructor.
             ):
                 self.add_message(
                     "missing-any-param-doc",
-                    args=(warning_node.name),
+                    args=warning_node.name,
```

Comment:
Could you revert this. This is building upon making `args` a tuple by default (one of my longer term projects).

Reply:
```suggestion
                    args=(warning_node.name,),
```
Shouldn't it be this instead then ?

Reply:
I think so!

Reply:
Done in https://github.com/PyCQA/pylint/pull/5575/commits/3cdadeb3c28d613db0e46f4d5e7d13ffc4f23c4b